.spi.yml

```diff
@@ -0,0 +1,19 @@
+# This is manifest file for the Swift Package Index for it to 
+# auto-generate and host DocC documentation.
+# For reference see https://swiftpackageindex.com/swiftpackageindex/spimanifest/documentation/spimanifest/commonusecases#Host-DocC-documentation-in-the-Swift-Package-Index.
+
+version: 1
+builder:
+  configs:
+    - documentation_targets: 
+      - IDEUtils
+      - SwiftBasicFormat
+      - SwiftCompilerSupport
+      - SwiftDiagnostics
+      - SwiftOperators
+      - SwiftParser
+      - SwiftParserDiagnostics
+      - SwiftRefactor
+      - SwiftSyntax
```

Comment:
@ahoppen Should `SwiftSyntax` be the "landing" target (i.e., the default target users will see when they visit this project's Documentation page on SwiftPackageIndex.com)? If so, this should be listed first.

Reply:
Yes, I think that would make sense. Thanks 🙏

Reply:
ok updated